modules/install-sno-ibm-z.adoc

```diff
@@ -0,0 +1,242 @@
+====
++
+The file names contain the {product-title} version number. They resemble the following examples:
+
+** kernel: `rhcos-<version>-live-kernel-<architecture>`
+** initramfs: `rhcos-<version>-live-initramfs.<architecture>.img`
+** rootfs: `rhcos-<version>-live-rootfs.<architecture>.img`
++
+[NOTE]
+====
+The rootfs image is the same for FCP and DASD.
+====
+
+. Move the downloaded {op-system-base} live `kernel`, `initramfs`, and `rootfs` as well as the Ignition files to an HTTP or HTTPS server.
+
+. Create parameter files for a particular virtual machine:
++
+.Example parameter file
++
+[source,terminal]
+----
+rd.neednet=1 \
+console=ttysclp0 \
+coreos.live.rootfs_url={rhcos_liveos}:8080/rootfs.img \ <1>
+ignition.config.url={rhcos_ign}:8080/ignition/bootstrap-in-place-for-live-iso.ign \ <2>
+ip=encbdd0:dhcp::02:00:00:02:34:02 <3>
+rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 \
+rd.dasd=0.0.4411 \ <4>
+rd.zfcp=0.0.8001,0x50050763040051e3,0x4000406300000000 \ <5>
+zfcp.allow_lun_scan=0 \
+rd.luks.options=discard \
+ignition.firstboot ignition.platform.id=metal \
+console=tty1 console=ttyS1,115200n8
+----
+<1> For `coreos.live.rootfs_url=`, specify the matching rootfs artifact for the `kernel`and `initramfs` you are booting. Only HTTP and HTTPS protocols are supported.
+<2> For `ignition.config.url=`, specify the Ignition file for the machine role. Only HTTP and HTTPS protocols are supported.
+<3> For ip=, assign the IP address automatically using DHCP or manually as described in "Installing a cluster with z/VM on {ibmzProductName} and {linuxoneProductName}".
+<4> For installations on DASD-type disks, use `rd.dasd=` to specify the DASD where {op-system} is to be installed. Omit this entry for FCP-type disks.
+<5> For installations on FCP-type disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {op-system} is to be installed. Omit this entry for DASD-type disks.
++
+Leave all other parameters unchanged.
+
+. Transfer the `initramfs`, `kernel`, parameter files, and {op-system} images to z/VM, for example by using FTP. For details about how to transfer the files with FTP and boot from the virtual reader, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/sect-installing-zvm-s390[Installing under Z/VM].
+
+. Punch the files to the virtual reader of the z/VM guest virtual machine that is to become your bootstrap node.
+
+. Log in to CMS on the bootstrap machine.
+
+. IPL the bootstrap machine from the reader by running the following command:
++
+----
+$ cp ipl c
+----
+
+. After firt reboot of the virtual machine, run the following commands directly after one another:
+
+.. To boot a DASD device after first reboot, run the following commands:
++
+[source,terminal]
+----
+$ cp i <devno> clear loadparm prompt
+----
++
+Where `<devno>` specifies the device number of the boot device as seen by the guest.
```

Comment:
```suggestion
[source,terminal]
----
$ cp i <devno> clear loadparm prompt
----
+
where:

`<devno>`:: Specifies the device number of the boot device as seen by the guest.
```

Reply:
See the repo guidelines about [Code formatting and syntax](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output). You have to scroll a bit, or just search the page for "where:"